test-toolkit/conftest.py

```diff
@@ -29,14 +28,33 @@
 logging.getLogger('auth.py').setLevel(logging.INFO)
 logging.getLogger('connectionpool.py').setLevel(logging.INFO)
 
-SCRIPT_PATH = os.path.dirname(os.path.realpath(__file__))
+DIR_PATH = os.path.dirname(os.path.realpath(__file__))
+
+# these regions have some p2 instances, but not enough for automated testing
+NO_P2_REGIONS = [
+    'ca-central-1',
+    'eu-central-1',
+    'eu-west-2',
+    'us-west-1',
+    'eu-west-3',
+    'eu-north-1',
+    'sa-east-1',
+    'ap-east-1',
+    'me-south-1'
+]
```

Comment:
do we still need this?

Reply:
no idea, probably, i don't remember p2 being added everywhere.

Reply:
I guess it's more a question of if these tests are being run in multiple regions since they're not part of releasing the images